DEVELOPMENT.md

```diff
@@ -42,17 +44,29 @@ integration environments), update the
 3. This updates the default [version mapping for the app in question](https://github.com/broadinstitute/terra-helmfile/blob/master/versions.yaml).
 4. [Our deployment of ArgoCD](https://ap-argocd.dsp-devops.broadinstitute.org/applications) monitors the above repo, and any environments in which the app is set to auto-sync will immediately pick up the new version of the image. If the app is not set to auto-sync in an environment, it can be manually synced via the ArgoCD UI or API.
 
-
 ## Setup
 
 ### Prerequisites:
 
-- Install Postgres 13.1: https://www.postgresql.org/download/
-  ```sh
-  brew install postgresql@13
-  ```
-  - [The app](https://postgresapp.com/downloads.html) may be easier, just make sure to download the right version. It'll manage things for you and has a useful menulet where the server can be turned on and off. Don't forget to create a server if you go this route.
-- Install AdoptOpenJDK Java 11 (Hotspot). Here's an easy way on Mac, using [jEnv](https://www.jenv.be/) to manage the active version:
+#### Postgres
+We are currently using Postgres 13.1.
+
+You have choices for how to install Postgres:
+- Directly from the Postgresql website: https://www.postgresql.org/download/
+- Via homebre:
+```sh
+brew install postgresql@13
+```
+- By installing the Mac application: [Postgresql app](https://postgresapp.com/downloads.html). This
+is the easiest. Just make sure to download the right version. It'll manage things for you and has a
+useful menulet where the server can be turned on and off. Don't forget to create a server if you
+go this route.
+
```

Comment:
There's also the Docker option, which I use. It's documented someplace...

Reply:
I reworded. It is discussed later in the document.